src/lib/containers/drug_upload_tool/DrugUploadForm.tsx

```diff
@@ -101,10 +102,23 @@ export default class DrugUploadForm extends React.Component<
       steps: steps,
       previousStepIds: [],
       formData: props.formData,
-      doShowErrors: false
+      doShowErrors: false,
+      doShowHelp: true
     };
   }
 
+  componentDidMount() {
+    if(_.isEmpty(this.state.formData)) {
+      const result = {};
+      const defs = this.props.schema.definitions;
+      Object.keys(defs).forEach((key: string) => {
+        if (defs[key].properties && Object.keys(defs[key].properties).indexOf('included') > -1) {
```

Comment:
How come this is required, this seems like something that would fall on the data curation side?